python/paddle/fluid/transpiler/distribute_transpiler.py

```diff
@@ -731,6 +741,15 @@ def _clone_var(block, var, persistable=True):
                 type="sum",
                 inputs={"X": table_grad_list},
                 outputs={"Out": [grad_var]})
+        else:
```

Comment:
In https://github.com/PaddlePaddle/Paddle/pull/10973/files#diff-3a23ecb6a98b666c84525f3f6fcd2bb6R729, `table_grad_list` maybe confused with `self. table_grad_list`, I think maybe better to store in different name, or seperate trainer side and server side transpile processes.

Reply:
follow comment, change to trainer_side_table_grad_list and pserver_side_table_grad_list